frontstage/controllers/auth_controller.py

```diff
@@ -51,3 +52,23 @@ def sign_in(username, password):
 
     bound_logger.info("Successfully signed in")
     return {}
+
+
+def delete_account(username: str):
+    bound_logger = logger.bind(email=obfuscate_email(username))
+    bound_logger.info("Attempting to delete account")
+    auth = "{}:{}".format(app.config["SECURITY_USER_NAME"], app.config["SECURITY_USER_PASSWORD"]).encode("utf-8")
```

Comment:
Why bother with all this faff?  All the other ones do something like 
```python
response = requests.delete(url, data=form_data, auth=app.config["BASIC_AUTH"])
```
without needing to spend so much time making auth headers.  Or is there something I'm missing?

Reply:
no may be you are right ... I just used the way it was used previously

Reply:
done!